runtime_builders/publish_builders.py

```diff
@@ -54,8 +58,11 @@ def main():
                         sys.exit(1)
                     full_prefix = prefix + project_name + '-'
                     latest_file = parts[0][len(full_prefix):]
-                    logging.info(latest_file)
+                    latest_file_with_project = latest[len(prefix):]
+                    manifest[project_name] = \
+                        {'target': {'file': latest_file_with_project}}
```

Comment:
You probably want to do something more like:

manifest[project_name][target][file] = latest_file_with_project (dealing with missing keys, of course)

For example, this would erase the deprecation section if there was one.  It might be appropriate to have this script write the entire "target" section though, rather than just updating the single file field.